src/main/java/com/syntaxphoenix/spigot/smoothtimber/SmoothTimber.java

```diff
@@ -48,8 +58,12 @@ public void onDisable() {
         if (PluginUtils.CHANGER == null || !PluginUtils.CHANGER.isValid()) {
             return;
         }
-        Bukkit.getScheduler().cancelTasks(this);
-        CooldownHelper.COOLDOWN.getTimer().kill();
+				if(IS_FOLIA) {
+					Bukkit.getServer().getGlobalRegionScheduler().cancelTasks(this);
+				} else {
+					Bukkit.getScheduler().cancelTasks(this);
+				}
+				CooldownHelper.COOLDOWN.getTimer().kill();
```

Comment:
Again formatting
1. Space after if
2. Spaces instead of tabs (4 spaces for each tab)